Jenkinsfile

```diff
@@ -109,27 +109,6 @@ pipeline {
                 }
             }
         }
-        stage('Security Analysis') {
-            parallel {
-                stage ('Owasp') {
```

Comment:
Are we no longer running owasp as a separate phase? I thought we didn't want to slow regular builds down too much?

Reply:
We're trying to move away from that when we can. This repo builds in less than 5 minutes on my laptop even with OWASP on.